src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java

```diff
@@ -253,9 +252,14 @@ public String getString2(final String component,
 		String message = "";
 		String source;
 		try {
-			ResourceBundle rb = ResourceBundle.getBundle(bundle,
-					LocaleUtility.defaultLocale);
-			source = rb.getString(key);
+			MessagesDTO dto = new MessagesDTO();
```

Comment:
How about this solution to load the bundle file to get source?
1 Remove 'bundle' parameter
2 Get the bundle value by component from the yaml config
3 Replace as:
 ResourceBundle rb = ResourceBundle.getBundle(**new_bundle**,
					LocaleUtility.defaultLocale)

Reply:
The bundle files may be json format, can't load them with this method.